llvm/utils/lit/tests/Inputs/googletest-no-sharding/DummySubDir/OneTest.py

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+if len(sys.argv) == 3 and sys.argv[1] == "--gtest_list_tests":
+    if sys.argv[2] != "--gtest_filter=-*DISABLED_*":
+        raise ValueError("unexpected argument: %s" % (sys.argv[2]))
```

Comment:
**Suggestion** Here and below, feel free to use f-strings since we require Python 3 nowadays.

Reply:
Done.